frontend/ui/src/app/tutorials/tutorials.component.html

```diff
@@ -0,0 +1,75 @@
+<section class="bg-gray-50 dark:bg-gray-900 p-3 sm:p-5 antialiased sm:ml-64">
+  <div class="mx-auto max-w-screen-lg px-4">
+    <div class="bg-white dark:bg-gray-800 relative shadow-md sm:rounded-lg overflow-hidden">
+      <div class="overflow-x-auto">
+        <ul role="list" class="divide-y divide-gray-200 dark:divide-gray-700">
+          @for (tutorial of tutorialsService.tutorialsProgress$ | async; track tutorial.id) {
+            <li class="py-3 px-6 sm:py-4">
+              <div class="flex items-center">
+                <div class="shrink-0 text-gray-900 dark:text-white">
+                  <fa-icon class="text-xl" [icon]="tutorial.icon"></fa-icon>
+                </div>
+                <div class="flex-1 min-w-0 ms-4">
+                  <a
+                    class="font-semibold text-gray-900 truncate dark:text-white hover:underline hover:text-blue-600 dark:hover:text-blue-500"
```

Comment:
Font shouldn't be semibold It looks to big in my opinion, regular font should be fine.